### DISCUSSION TITLE
POCO Series

### DISCUSSION THREAD

**Discussion by R9TRH** [R9TRH/MIUIwidgets#5] [General]
V6.x: Alpha Launcher
V7.x: EU launcher
V7.x-Alter: for MIUI 13 ANDROID 12

Flash anti low end for full functions 

<hr /><em>This discussion was created from the release <a href='https://github.com/R9TRH/MIUIwidgets/releases/tag/v1.3'>POCO Series</a>.</em>

**Comment by R9TRH** [comment]
Poco users give feedback :)

**Reply by henriks79** [reply]
I tested the alternative MW 7.4 and 7.5 in poco F3 European, but the problem remains, the miui launcher does not install, only the poco launcher appears, the rest of the applications such as vault, themes and apk time, install ok

**Reply by R9TRH** [reply]
Use Only Poco Series. Try V6, if still problem remains, tell me your rom version I'll take a look at it and try to fix
